pkg/snmp/snmp_test.go

```diff
@@ -526,6 +527,8 @@ network_devices:
 	assert.Equal(t, "legacyCommunityString", legacyConfig.Community)
 	assert.Equal(t, "legacySnmpVersion", legacyConfig.Version)
 	assert.Equal(t, "127.2.0.0/30", legacyConfig.Network)
+	warnings := pkgconfigsetup.Datadog().Warnings()
```

Comment:
Instead of using the global `pkgconfigsetup.Datadog()` you can change line 505 to this:

```
corecfg := configmock.NewFromYAML(t, `
```

Then here use:

```
warnings := corecfg.Warnings()
```